src/helpers/sharingHelper.js

```diff
@@ -253,7 +253,7 @@ module.exports = {
     return httpHelper
       .getOCS(apiURL, user)
       .then((res) => {
-        httpHelper.checkStatus(res, 'The response status is not the expected value')
+        httpHelper.checkStatus(res, 'The HTTP response status is not the expected value in response to getting all shares.')
```

Comment:
I am confused by this message!
```suggestion
        httpHelper.checkStatus(res, 'The HTTP response status is not the expected value')
```

Reply:
@sawjan dai **The HTTP response status is not the expected value**  this message is the same for all. Hard to differentiate the request type